src/viam/sdk/spatialmath/geometry.cpp

```diff
@@ -170,22 +170,22 @@ viam::common::v1::Geometry GeometryConfig::to_proto() const {
     }
 }
 void GeometryConfig::set_coordinates(coordinates coordinates) {
-    pose_.coordinates = std::move(coordinates);
+    pose_.coordinates = coordinates;
 }
 void GeometryConfig::set_pose(pose pose) {
-    pose_ = std::move(pose);
+    pose_ = pose;
 }
 void GeometryConfig::set_geometry_specifics(geometry_specifics gs) {
     geometry_specifics_ = std::move(gs);
 }
 void GeometryConfig::set_pose_orientation(pose_orientation orientation) {
-    pose_.orientation = std::move(orientation);
+    pose_.orientation = orientation;
 }
 void GeometryConfig::set_theta(double theta) {
-    pose_.theta = std::move(theta);
+    pose_.theta = theta;
 }
 void GeometryConfig::set_geometry_type(GeometryType type) {
-    geometry_type_ = std::move(type);
+    geometry_type_ = type;
```

Comment:
These all look like should be fine to move. What is the complaint?

Reply:
These types are all trivially copyable, so the move has no effect.